services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return (Movie.objects.filter(genres__in=genres_ids)
+                .filter(actors__in=actors_ids))
```

Comment:
When filtering by both genres and actors, the current implementation uses two consecutive `filter` calls. This will return movies that match all specified genres and actors, which may not be the intended behavior if you want movies that match any of the criteria. Consider using `Q` objects with `|` (OR) to allow more flexible filtering.